src/js/main.js

```diff
@@ -4,19 +4,31 @@ import share from './lib/share';
 import sheetToDOM from './lib/sheettodom';
 import emailsignupURL from './lib/emailsignupURL';
 
-var shareFn = share('Interactive title', 'http://gu.com/p/URL', '#Interactive');
+// TODO remove once information is in sheet
+const tempSocialConfig = {
+    title: 'Interactive title',
+    url: 'http://gu.com/p/URL',
```

Comment:
I believe not anymore, as twitter auto shortens them using t.co, Frontend no longer uses the `shortUrl` see https://github.com/guardian/frontend/pull/13568